### PR TITLE
feat: mobile bottom nav

### PR DESCRIPTION
**Pull request by lxhughes**
https://app.asana.com/0/1200180014510248/1202131476892167/f

Adds a bottom bar on mobile with 4 links: ladder page, maps, search, and swings view. Tabs entry point to come later.

### CODE REVIEW

File: assets/css/nav/_bottom_nav_mobile.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+.m-bottom-nav-mobile__link,
+.m-bottom-nav-mobile__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  padding: 0.75rem;
+  border-radius: 0.375rem;
+  font-weight: 400;
```
Comment (lemald) [inline assets/css/nav/_bottom_nav_mobile.scss:22]: Is the `font-weight` necessary here? I thought there was no text in the bottom nav. Same question applies for the `active` class below.

File: assets/css/nav/_bottom_nav_mobile.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    svg {
+      fill: $color-eggplant-800;
+    }
+  }
+}
+
+.m-bottom-nav-mobile__icon {
+  &.m-bottom-nav-mobile__icon--swings-view svg {
+    transform: scaleX(-1);
```
Comment (lemald) [inline assets/css/nav/_bottom_nav_mobile.scss:55]: Out of scope for this ticket (and I know I'm the one who introduced this with the desktop left nav), but when we go and redo a bunch of our SVG handling we should definitely just replace the swings SVG with one that's pointing the right way around.

**Review by lemald — CHANGES_REQUESTED** [review]
Left a couple of minor comments, but looks good overall. 👍

File: assets/css/_nav.scss [hunk 3]
```diff
@@ -17,7 +17,8 @@
   grid-template-rows: max-content minmax(0, 1fr);
   grid-template-areas:
     "top top"
-    "left content";
+    "left content"
+    "bottom bottom";
```
[the remark this answers is not in the document]
Comment (lxhughes) [reply assets/css/_nav.scss]: fixed grid in 6e8e265

File: assets/css/nav/_bottom_nav_mobile.scss (hunk 1 above)
Comment (lxhughes) [reply assets/css/nav/_bottom_nav_mobile.scss:22]: resolved in 9868b3c

File: assets/css/_nav.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (lxhughes) [reply assets/css/_nav.scss]: *actually fixed grid in 5e251dc

**Review by lemald — APPROVED** [review]
Looks good to me pending product / design review.

**Pull request merged**